PocketMC.RemoteControl/Web/index.html

```
@@ -276,16 +422,121 @@ <h2>Console</h2>
+          </div>
+
+          <div id="noBackupsMsg" class="empty-state-card" hidden>
+            <div class="empty-state-icon">
+              <svg viewBox="0 0 24 24"><path d="M19 21H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h11l5 5v11a2 2 0 0 1-2 2z"></path></svg>
+            </div>
+            <h4 class="empty-state-title">No Backups Found</h4>
+            <p class="empty-state-desc">Click "Create Backup Now" to create your first server backup zip.</p>
+          </div>
+        </div>
+      </section>
+      </section>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </section> ]

(tag-pair)